src/components/footer/footer.tsx

```diff
@@ -0,0 +1,116 @@
+import Link from 'next/link';
+
+export const Footer = () => {
+  const currentYear = new Date().getFullYear();
+  return (
+    <footer className="items-center justify-evenly bg-gray-300 py-4 text-center text-black lg:flex">
+      <Link href="https://www.youtube.com/@tapasadhikary" target="_blank">
+        &copy; {currentYear} An Open Source initiative from tapaScript Community.
+      </Link>
+      <div className="mt-4 flex justify-center gap-4 lg:mt-0 lg:gap-8">
+        {SOCIAL_LINKS.map((item, id) => (
+          <Link key={id} href={item.link} target="_blank">
+            <div className=" hover:scale-105">{item.icon}</div>
+          </Link>
+        ))}
+      </div>
+    </footer>
+  );
+};
+
+const ICONS = {

```

Comment:
can this be taken to a different file like `src/config/site.ts`

Reply:
I have handled the icons differently now. These are no more hard coded SVGs.. so removed this. Resolving.